### PR TITLE
Update local-objects-classes.adoc

### PR DESCRIPTION
**Pull request by christyjacob4**
Split the code examples into multiple lines to align with the Motoko Style guides.

### CODE REVIEW

File: doc/modules/language-guide/pages/local-objects-classes.adoc [hunk 1]
```diff
@@ -353,7 +365,7 @@ The constituents of the body marked `public` contribute to the resulting objects
 
 =====  Another example: `Bits`
 
-As another example, let's consider the task of walking the bits of a `Nat`ural number. For this example, we could define the following:
+As another example, let's consider the task of walking the bits of a Natural number. For this example, we could define the following:
```
Comment (christyjacob4) [inline doc/modules/language-guide/pages/local-objects-classes.adoc:368]: Before:
Screenshot 2021-12-11 at 3 18 03 PM

After:

Comment (nomeata) [reply doc/modules/language-guide/pages/local-objects-classes.adoc:368]:
```suggestion
As another example, let's consider the task of walking the bits of a natural number (type `Nat`). For this example, we could define the following:
```

Comment (nomeata) [reply doc/modules/language-guide/pages/local-objects-classes.adoc:368]: Host about this?

Comment (christyjacob4) [reply doc/modules/language-guide/pages/local-objects-classes.adoc:368]: I think that's a great suggestion 👍

**Review by nomeata — APPROVED** [review]

**Pull request merged**